test/exercism/use_cases/attempt_test.rb

```diff
@@ -90,11 +92,11 @@ def test_a_new_attempt_supersedes_the_previous_one
 
   def test_a_new_attempt_supersedes_the_previous_hibernating_one
     submission = Submission.create(user: user, code: 'CODE 1', language: 'ruby', slug: 'two', created_at: Time.now, state: 'hibernating')
-    Attempt.new(user, Iteration.new({'ruby/two/two.rb' => 'CODE 2'}, {track: 'ruby', slug: 'two '})).save
+    Attempt.new(user, Iteration.new({'ruby/two/two.rb' => 'CODE 2'}, {track: 'ruby', slug: 'two'})).save
```

Comment:
Line is too long. [104/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.